packages/quantic/cypress/integration/results-per-page/results-per-page.cypress.ts

```diff
@@ -27,8 +29,12 @@ describe('quantic-result-per-page', () => {
       },
     });
     configure(options);
+    if (options.useCase === useCaseEnum.insight) {
```

Comment:
So why exactly do we wait only whenever the useCase is insight?

Reply:
It is because the headless bundle has to be downloaded before we can execute the first search query. We don't have this problem with the search use case because the QuanticSearchInterface is triggering the first query automatically. For the insight use case, the first search has to be triggered by the renderer component, not from Quantic.

It would be nice though if the QuanticInsightInterface component could have some sort of marker to inform the test framework that the component is properly initialized.

Reply:
@lbergeron  I create a hidden input with attribute `is-initialized` to replace the `cy.wait(10000`